quicfire_tools/examples.py

```diff
@@ -46,7 +47,9 @@ def create_line_fire() -> SimulationInputs:
     sim_inputs.quic_fire.ignitions_per_cell = 100
 
     # Write the input file deck to a directory
-    sim_inputs.write_inputs("/tmp")
+    qf_path = Path(__file__).parent / "linefire_example"
+    qf_path.mkdir(parents=True, exist_ok=True)
+    sim_inputs.write_inputs(qf_path)
```

Comment:
I think I put this stuff in here to compare it with the linefire example and forgot to take it out. The way that I saw examples going is that this function would return sim_inputs, and then a user could import the linefire simulation code like:

```python
from quicfire_tools.examples import line_fire_simulation
```

The problem with including this writing code is that if a user runs `import quicfire_tools.examples` then the simulation files will all get written to the package directory and the user will have no way to access them.

Reply:
That makes sense. If that is the purpose of examples.py, then I agree it shouldn't write the files automatically. I think though that Zach and I had a different thing in mind altogether that we should maybe do in addition to examples.py. We'll chat in slack